core/market/src/matcher.rs

```diff
@@ -109,9 +112,31 @@ impl Matcher {
         tokio::task::spawn_local(cyclic::bcast_offers(self.clone()));
         tokio::task::spawn_local(cyclic::bcast_unsubscribes(self.clone()));
 
+        self.bind_neighbourhood_bcast(local_prefix).await.ok();
+
         self.bind_expiration_tracker()
             .await
             .map_err(|e| MatcherInitError::ExpirationTrackerError(e.to_string()))?;
+
+        Ok(())
+    }
+
+    async fn bind_neighbourhood_bcast(&self, local_prefix: &str) -> anyhow::Result<()> {
+        let bcast_address = format!("{local_prefix}/{}", NewNeighbour::TOPIC);
+        let myself = self.clone();
+        bind_broadcast_with_caller(
+            &bcast_address,
+            move |_caller, _msg: SendBroadcastMessage<NewNeighbour>| {
+                let myself = myself.clone();
+                async move {
+                    info!("Received new neighbour broadcast");
```

Comment:
I would log caller, to know from which Node this broadcast came. And probably we could lower log level now, since it is not so important.